src/earthkit/workflows/__init__.py

```diff
@@ -8,6 +8,7 @@
 
 import dill
 
+from ._version import __version__
```

Comment:
I would catch an exception here and default to a generic version "version_not_found" or something, in case people are playing with the package without installing it through pip. See for instance:
https://github.com/ecmwf/earthkit-data/blob/develop/src/earthkit/data/__init__.py